init.py

```diff
@@ -70,29 +97,11 @@
             await message.channel.send(f'Account {nickname} already exists')
             return
 
-        newTaskAccount = taskAccount(spreadsheetUrl, nickname, False)
-        try:
-            sheet.updateTaskAccount(newTaskAccount)
-        except Exception as e:
-            await message.channel.send(f'Error: {e}')
-            return
-        taskAccountList.add(newTaskAccount)
-        taskAccountList.updateLastUpdated(newTaskAccount, time())
-        await message.channel.send(f'Added account "{nickname}"')
-    elif(command == 'addofficial'):
-        if(len(result) < 3):
-            await message.channel.send('Error: You need to add task accounts with the syntax !addofficial [[spreadsheetUrl]] [[nickname]]')
-            return
-        spreadsheetId = result[1].strip()
-        nickname = result[2]
-        for res in result[3:]:
-            nickname += ' ' + res
-        
-        if(getTaskAccountFromNickname(nickname)):
-            await message.channel.send(f'Account {nickname} already exists')
-            return
+        if (isOfficial):
+            newTaskAccount = taskAccount(spreadsheetUrl, nickname, True)
+        else
+            newTaskAccount = taskAccount(spreadsheetUrl, nickname, False)
```

Comment:
No need for if...else

`newTaskAccount = taskAccount(spreadsheetUrl, nickname, isOfficial)`